### ISSUE TITLE
Idea to bring start page as non modal window in VS 2019

### ISSUE THREAD

**Issue by SylwesterZarebski** [Danielku15/BetterStartPage#44]
I recently found extension https://marketplace.visualstudio.com/items?itemName=JanKucera.StartPageOnStartup, which bring back original start page and maybe it is possible to write custom page similar to this extension?

**Comment by Danielku15** [comment]
This extension leverages the fact that Microsoft still ships the old classical Start Page code somewhere hidden. This extension loads the old start page and simply triggers the command to launch the start page. 

It is very likely that Microsoft will soon remove this legacy code. There it would be more realistic to simply create a full own start page window.

**Comment by SylwesterZarebski** [comment]
Yes, but start window is modal, which is less useful for me than start page - you can't interact with VS using window, have to close, and then is no real way (besides of closing VS and reopen) to bring this window back.
Maybe it is possible to create custom page... i dig this issue deeper...

**Comment by MagicAndre1981** [comment]
> It is very likely that Microsoft will soon remove this legacy code. 

and they did this in 16.3 Preview 3
